dispatcher.go

```diff
@@ -201,12 +207,12 @@ func (d *Dispatcher) Inbounds() Inbounds {
 //
 // 	keyvalueClient := json.New(dispatcher.ClientConfig("keyvalue"))
 //
-// This function panics if the service name is not known.
-func (d *Dispatcher) ClientConfig(service string) transport.ClientConfig {
-	if rs, ok := d.outbounds[service]; ok {
-		return clientconfig.MultiOutbound(d.name, service, rs)
+// This function panics if the outboundKey is not known.
```

Comment:
I agree we should rename this term to `OutboundKey` - let's be sure we rename this everywhere.